go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/NearNodeFlash/nnf-integration-test
 go 1.19
 
 require (
```

Comment:
I wonder if it'll be happy if you add a collection of 'replace' lines after line 4:

```
replace github.com/DataWorkflowServices/dws => ../dws
replace github.com/NearNodeFlash/lustre-fs-operator => ../lustre-fs-operator
...
```

Reply:
But then it can't be used outside of an nnf-deploy workarea, so scratch that.